Makefile

```diff
@@ -71,6 +71,12 @@ build-js: $(node_modules) clean-js build-buf
 pack: build
 	npm pack
 
+# bump to the next pre-release version on the `next` distribution tag -
+# run this command if you need to publish an off-cycle release of the sdk
+.PHONY: bump-next-version
+bump-next-version:
+	npm version --no-git-tag-version prerelease --preid=next
```

Comment:
`next` versions will always be patch versions with `-next.X` suffixes. this might be a little awkward if next stable release ends up being a major or minor bump, since it will signal a patch bump that will next be released, but all in all not the worst situation